scripts/test_wheel.sh

```diff
@@ -114,7 +156,16 @@ $PYTHON -c "import sys; lines=sys.stdin.read(); \
 cd "$TEST_DIR"
 
 # run tests
-${PYTHON} -m pytest --cov -v --durations=100 \
-  --junit-xml="$WORKSPACE"/pytest.xml
+if [[ "$USE_MINIMAL" -eq 1 ]]; then
+  # remove all toolkits related files
+  minimal_test=($(grep -L "toolkits" *.py))
```

Comment:
I think selecting which tests to run in this is going to be very brittle. You could be excluding a test just because it uses the word "toolkits" in a comment.

I think it would be better to [custom pytest markers](https://docs.pytest.org/en/latest/example/markers.html).

Reply:
Here you go. https://github.com/apple/turicreate/pull/3171